.gitignore

```diff
@@ -25,4 +25,5 @@ dist-ssr
 
 # E2E tests
 screenshots
-videos
\ No newline at end of file
+videos
+docker
```

Comment:
It would be quite surprising to someone to make a directory called `docker` and it to be gitignored.
What else is in `docker` besides `screenshots` and `videos` already here?